Assets/HoloToolkit/Input/Tests/Scripts/FocusedObjectMessageReceiver.cs

```diff
@@ -0,0 +1,31 @@
+﻿using UnityEngine;
+using System.Collections;
+
+/// <summary>
+/// FocusedObjectMessageReceiver class shows how to handle messages sent by FocusedObjectMessageSender.
+/// This particular implementatoin controls object appearance by changing its color when focused.
+/// </summary>
+public class FocusedObjectMessageReceiver : MonoBehaviour
```

Comment:
We went back and forth on this in #69, the reason for refactoring this into a separate class is that developers will want different ways of handling object selection, not necessarily based on gaze, so it feels more natural to keep the send message part out of GazeManager, especially that there is also a SelectedObjectMessageSender, which shows a different way to select objects. Selection can also be via airtap, alone or in addition to voice commands.